lib/config/validation.spec.ts

```diff
@@ -692,9 +692,13 @@ describe('config/validation', () => {
         true,
       );
       expect(warnings).toHaveLength(0);
-      expect(errors).toHaveLength(1);
+      expect(errors).toHaveLength(2);
```

Comment:
In a future change, we could remove the existing validation if we wanted to (albeit I prefer the more specific message `Invalid customType: unknown. Key is not a custom manager`)